.github/workflows/test.yml

```diff
@@ -116,7 +116,7 @@ jobs:
         with:
           payload: |
             {
-              "test": "testNode${{ matrix.node }}"
+              "test": "A 'testNode${{ matrix.node }}' task has failed."
```

Comment:
Original slack-webhook message body was configured to run only for failures. I updated the configuration to make it more generic.
So we now need to pass in the complete message we intend to post on slack.